platform-api/che-core-api-builder/src/main/java/org/eclipse/che/api/builder/internal/SourcesManagerImpl.java

```diff
@@ -202,8 +205,6 @@ public void write(byte[] b) throws IOException {
     };
 
     private void download(String downloadUrl, java.io.File downloadTo, File directory) throws IOException {
-        HttpURLConnection conn = null;
-        try {
             final LinkedList<java.io.File> q = new LinkedList<>();
```

Comment:
@garagatyi I did this to avoid bumping the whole file to the left and affect only the specific lines that changed. I can un-indent it.